WordPress/src/main/java/org/wordpress/android/ui/people/PeopleInviteFragment.java

```diff
@@ -423,6 +431,10 @@ private void checkAndSend() {
             return;
         }
 
+        if (!NetworkUtils.checkConnection(getActivity())) {
+            return;
```

Comment:
Aha, that's true! Forgot about that toast. Sorry for the unneeded proposal.

BTW, notice that when no connection, `checkAndSend()` doesn't get called, and prolly this is part of the root cause for #4332